### PR TITLE
replaces fixed sortedm2m list width

### PR DESCRIPTION
**Pull request by lory87**
**This pull request fixes the following issues**:

|BEFORE|AFTER|
|---|---|
|screen shot 2016-01-04 at 14 34 34|screen shot 2016-01-04 at 14 36 08|

### CODE REVIEW

File: djangocms_admin_style/sass/components/_content.scss [hunk 1]
```diff
@@ -288,6 +288,11 @@ ul.messagelist + #content {
 // filter horizontal m2m
 .sortedm2m-container {
     width: 100% !important;
+    // make sure that list items don't have a fixed width to not break mobile view #282
+    ul.sortedm2m,
```
Comment (dhorelik) [inline djangocms_admin_style/sass/components/_content.scss]: is `.sortedm2m` not enough in terms of specificity?

Comment (lory87) [inline djangocms_admin_style/sass/components/_content.scss]: not unfortunately not because I have to override the widget

Comment (dhorelik) [inline djangocms_admin_style/sass/components/_content.scss]: ok then

**Comment by dhorelik** [comment]
lgtm

**Comment by FinalAngel** [comment]
LGTM

**Pull request merged**